unified-codes/apps/legacy-api/src/app/types.ts

```diff
@@ -0,0 +1,23 @@
+import * as Http from 'http';
```

Comment:
We have two fastify servers, so might be useful to put this in the `data` lib to share between `legacy-api` and `data-service`?